docs/add-ins/metadata-for-an-outlook-add-in.md

```diff
@@ -273,6 +273,54 @@ GET https://outlook.office.com/api/v2.0/Me/Events?$filter=SingleValueExtendedPro
 
 For other examples that use REST to get single-value MAPI-based extended properties, see [Get singleValueExtendedProperty](/graph/api/singlevaluelegacyextendedproperty-get?view=graph-rest-1.0).
 
+The following example shows how to get an item and its custom properties. In the callback function for the `done` method, `item.SingleValueExtendedProperties` will give a list of the requested custom properties.
```

Comment:
"will give a list of the requested custom properties." -> "contains a list of the requested custom properties."